jax/_src/lru_cache.py

```diff
@@ -125,22 +133,32 @@ def put(self, key: str, val: bytes) -> None:
       warnings.warn(msg)
       return
 
-    file = self.path / key
+    cache_file = self.path / f"{key}{self._cache_suffix}"
+    atime_file = self.path / f"{key}{self._atime_suffix}"
 
     if self.eviction_enabled:
       self.lock.acquire(timeout=self.lock_timeout_secs)
 
     try:
-      if file.exists():
+      if cache_file.exists():
         return
 
       self._evict_if_needed(additional_size=len(val))
-      file.write_bytes(val)
+
+      cache_file.write_bytes(val)
+
+      # update timestamp
+      timestamp = self._get_current_timestamp()
+      atime_file.write_text(str(timestamp))
 
     finally:
       if self.eviction_enabled:
         self.lock.release()
 
+  @staticmethod
+  def _get_current_timestamp() -> int:
```

Comment:
I would personally just inlined this. Why have a method?

Reply:
Agree. Done